src/worker.cpp

```diff
@@ -1393,4 +1393,99 @@ void Worker::performBackupAndRestore(types::VPDMapVariant& io_srcVpdMap)
         */
     }
 }
+
+void Worker::deleteFruVpd(const std::string& i_dbusObjPath)
+{
+    try
+    {
+        if (m_parsedJson.empty())
```

Comment:
Why JSON should be mandatory for deletion?

Reply:
Updated